docs/apis-tools/testing/getting-started.md

```diff
@@ -0,0 +1,290 @@
+
+- Zeebe
+- Operate
+- Tasklist
+- Connectors
+
+:::warning Disclaimer
+For Camunda 8.6, CPT is in an [alpha version](/reference/alpha-features.md#alpha) (i.e. preview).
+
+If you need a full-featured testing library, take a look at [Zeebe Process Test](/apis-tools/java-client/zeebe-process-test.md).
+:::
+
+:::note Limitations
+CPT is in an early stage. It doesn't contain all features, and it is not optimized yet. Please be aware of the following limitations:
+
+- Slow test execution (~ 40 seconds per test case)
+- Only basic assertions
+
+:::
+
+## Prerequisites
+
+- Java 8+ / 17+ (for Spring SDK)
+- JUnit 5
+- A Docker-API compatible container runtime, such as Docker on Linux or Docker Desktop on Mac and Windows. If you have issues with your Docker runtime, have a look at the [Testcontainers documentation](https://java.testcontainers.org/supported_docker_environment/).
+
+## Install
+
+We have two variations of CPT: for the [Spring SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) and the [Zeebe Java client](/apis-tools/java-client/index.md). Choose the one depending on which library you use in your process application.
+
+Add the following dependency to your Maven project:
+
+<Tabs groupId="client" defaultValue="spring-sdk" queryString values={
+[
+{label: 'Spring SDK', value: 'spring-sdk' },
+{label: 'Java client', value: 'java-client' }
+]
+}>
+
+<TabItem value='spring-sdk'>
+
+```xml
+<dependency>
+  <groupId>io.camunda</groupId>
+  <artifactId>camunda-process-test-spring</artifactId>
+  <scope>test</scope>
+</dependency>
+```
+
+</TabItem>
+
+<TabItem value='java-client'>
+
+```xml
+<dependency>
+  <groupId>io.camunda</groupId>
+  <artifactId>camunda-process-test-java</artifactId>
+  <scope>test</scope>
+</dependency>
+```
+
+</TabItem>
+
+</Tabs>
+
+## Write a test
+
+Create a new Java class with the following structure:
+
+<Tabs groupId="client" defaultValue="spring-sdk" queryString values={
+[
+{label: 'Spring SDK', value: 'spring-sdk' },
+{label: 'Java client', value: 'java-client' }
+]
+}>
+
+<TabItem value='spring-sdk'>
+
+```java
+package com.example;
+
+import io.camunda.process.test.api.CamundaAssert;
+import io.camunda.process.test.api.CamundaProcessTestContext;
+import io.camunda.process.test.api.CamundaSpringProcessTest;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+@CamundaSpringProcessTest
+public class MyProcessTest {
+
+    @Autowired private ZeebeClient client;
+    @Autowired private CamundaProcessTestContext processTestContext;
+
+    @Test
+    void shouldCompleteProcessInstance() {
+        // given
+        client.newDeployResourceCommand().addResourceFromClasspath("my-process.bpmn").send().join();
```

Comment:
🔧 I wonder if this could confuse Spring users as they likely use the auto deployment feature and don't need to explicitly deploy in their test case. WDYT about removing this line, or adding a comment that it's not necessary if the spring SDK already deploys your resources.